### DISCUSSION TITLE
Supermassive Stars/Population III Stars

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#124] [Astrophysics and Physics]
Hey everyone, came across this video: https://www.youtube.com/watch?v=dy8ZgD2Usu4&pp=ygUUc3R1bXBlZCBjb3Ntb2xvZ2lzdHM%3D and my first thought was that what it was describing could be the long theorized Population III Stars that are supposed to have existed in the early years of a Big Bang universe. What are all your thoughts on the video and the reported discovery?

**Comment by RedshiftDrift** [comment]
It is unlikely that these bright objects are Pop III stars because of the observed high N/O ratio.  In theory, Pop III stars should have an extremely low metallicity.  They would be too young to have nitrogen in such large quantities.

Note that the video proposes many scenarios since they have no clue what's going on.  Also, there are a large number of  *uncertain affirmations* such as "**could** explain...", "**may** evolve...", "**if** astronomers detect this spectral line...", etc., which have not been confirmed.

This video continues in the latest trend of discoveries of huge objects at large distances, e.g. the star [Earendel](https://en.wikipedia.org/wiki/WHL0137-LS), [too many massive black holes/too much of carbon molecules in the early universe](https://youtu.be/Y8du5wXbt90), galaxies that are too bright, and all the "supposedly extreme objects" discovered by JWST.  The stars are so big/bright they all seem to violate the [Eddington limit](https://en.wikipedia.org/wiki/Eddington_luminosity) (the maximum speed at which stars can form).  Sure, there can be exceptions like massive black holes, galactic collisions, Wolf-Rayet stars, etc... but when every object is an exception there must be some major error with the underlying model (in this case, Big Bang cosmology).

To astronomers who analyze observations with the Big Bang model, the few million years available make these objects appear as monsters with extreme properties.  Astronomers and cosmologists are forced to make up different excuses for each observation that doesn't fit the model.  Of course, the simple explanation of all this is that space does not expand, the universe had no beginning (and won't end), and all these galaxies are just like the ordinary galaxies we see closer to us without Pop III stars.
